src/5_with_chain.md

```diff
@@ -27,7 +27,7 @@ Building the chain might take up to 30 min.
 ### Start the chain
 
 ```bash
-./target/release/node-portablegabi --dev
+./target/debug/node-portablegabi --dev
```

Comment:
maybe replace this with `cargo run -- --dev`. The binary is placed in release or debug depending on your build...